src/main/java/wellnus/focus/feature/Session.java

```diff
@@ -6,18 +6,32 @@
 /**
  * Represents a session of Countdown objects.
  * A session is a sequence of Countdown objects represented by an ArrayList.
+ * We define a Session to have 4 states.
+ * <ol>
+ *     <li> Ready
+ *          `config` and `start` are only legal here.
+ *     <li> Counting
+ *          `pause` and `stop` are only legal here.
+ *     <li> Waiting
+ *          `next` and `stop` are only legal here.
+ *     <li> Paused
+ *          `resume` and `stop` are only legal here.
+ * </ol>
+ * The last timer holds a special `isReady` attribute to help Session determine if `start` and `config` is usable.
  */
+//@@author YongbinWang
 public class Session {
     private static final int INCREMENT = 1;
-    private final ArrayList<Countdown> session;
+    private static final boolean IS_LAST_COUNTDOWN = true;
     private String workDescription = "Task";
+    private final ArrayList<Countdown> session;
     private String breakDescription = "Break";
     private String longBreakDescription = "Long Break";
     private int work = 1;
     private int brk = 1;
     private int longBrk = 1;
     private int cycle = 2;
```

Comment:
Can consider changing to static final constant named: DEFAULT_WORK_DESCRIPTION etc.